js/components/matrix.js

```diff
@@ -0,0 +1,363 @@
+Fliplet.FormBuilder.field('matrix', {
+  name: 'Matrix',
+  category: 'Multiple options',
+  props: {
+    description: {
+      type: String
+    },
+    value: {
+      type: String
+    },
+    rowOptions: {
+      type: Array,
+      default: function() {
+        return [
+          {
+            label: T('widgets.form.matrix.defaultOptions.row1')
+          },
+          {
+            label: T('widgets.form.matrix.defaultOptions.row2')
+          }
+        ];
+      }
+    },
+    columnOptions: {
+      type: Array,
+      default: function() {
+        return [
+          {
+            label: T('widgets.form.matrix.defaultOptions.column1')
+          },
+          {
+            label: T('widgets.form.matrix.defaultOptions.column2')
+          }
+        ];
+      }
+    }
+  },
+  watch: {
+    value: function(val) {
+      if (this.checkValue(val) === 'clear') {
+        this.clearValue();
+      } else {
+        this.setValue();
+      }
+    }
+  },
+  methods: {
+    /**
+     * Returns unique ID to the form widget instance, field, row and column.
+     *
+     * @param {Number} rowIndex - an row index
+     *
+     * @param {Number} columnIndex - an column index
```

Comment:
Please check everything for correct English spelling and grammar

```suggestion
     * @param {Number} columnIndex - a column index
```